MapboxNavigation/NavigationView.swift

```diff
@@ -193,11 +193,10 @@ open class NavigationView: UIView {
         let informationChildren: [UIView] = [instructionsBannerView, lanesView, nextBannerView, statusView]
         informationStackView.addArrangedSubviews(informationChildren)
         
-        let constraints = informationChildren.flatMap { (view) -> [NSLayoutConstraint] in
-            return [view.leadingAnchor.constraint(equalTo: informationStackView.leadingAnchor),
-                    view.trailingAnchor.constraint(equalTo: informationStackView.trailingAnchor)]
+        informationChildren.forEach {
```

Comment:
Suggestion: I would caution against setting `isActive = true` in `forEach` closures. I've been burned by this in the past, although I don't know why it occurs. Generally if I'm activating more than one constraint at a time I try to use `NSLayoutConstraint.activate(_:)`.

Reply:
It's especially important to activate/deactive
 conflicting constraints simultaneously for example when switching between size classes. However, activating one by one through loadView, viewDidLoad, or initWithFrame is totally safe.